.github/dependabot.yml

```diff
@@ -1,11 +1,13 @@
-# Keep compatible webpack@4.x - Reference: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates
 
```

Comment:
To clean up dependabot auto-PR creation, we should raise them monthly, and ignore breaking major upgrades, since we will ignore anyways.